frontend/workflows/dynamodb/src/update-capacity.tsx

```diff
@@ -0,0 +1,254 @@
+import React from "react";
+import type { clutch as IClutch } from "@clutch-sh/api";
+import {
+  Button,
+  ButtonGroup,
+  CheckboxPanel,
+  client,
+  MetadataTable,
+  Resolver,
+  Table,
+  TableRow,
+  useWizardContext,
+} from "@clutch-sh/core";
+import { useDataLayout } from "@clutch-sh/data-layout";
+import type { WizardChild } from "@clutch-sh/wizard";
+import { Wizard, WizardStep } from "@clutch-sh/wizard";
+import styled from "@emotion/styled";
+import _ from "lodash";
+import { number } from "yup";
+
+import type { ResolverChild, WorkflowProps } from "./index";
+
+const Container = styled.div({
+  display: "flex",
+});
+
+const TableIdentifier: React.FC<ResolverChild> = ({ resolverType }) => {
+  const { onSubmit } = useWizardContext();
+  const resolvedResourceData = useDataLayout("resourceData");
+
+  const onResolve = ({ results }) => {
+    // Decide how to process results.
+    resolvedResourceData.assign(results[0]);
+    onSubmit();
+  };
+
+  return <Resolver type={resolverType} searchLimit={1} onResolve={onResolve} />;
+};
+
+const TableDetails: React.FC<WizardChild> = () => {
+  const { onSubmit, onBack } = useWizardContext();
+  const resourceData = useDataLayout("resourceData");
+  const capacityUpdates = useDataLayout("capacityUpdates");
+  const table = resourceData.displayValue() as IClutch.aws.dynamodb.v1.Table;
+
+  const handleTableCapacityChange = (key: string, value: string) => {
+    const newTableCapacity = { ...capacityUpdates.displayValue().table_throughput };
+    newTableCapacity[key] = value;
```

Comment:
nit simplify and confusing var name change
```suggestion
    const newTableThroughput = { ...capacityUpdates.displayValue().table_throughput, key: value };
```